static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -271,6 +271,27 @@ $block-padding-right: $gs-gutter;
 
     .block-elements {
 
+        /* Bylines
+           ========================================================= */
+        .liveblog-block-byline {
+            border-bottom: 1px dotted #dcdcdc;
+            padding-bottom: 8px;
```

Comment:
Yeah @robertberry is right - we use those two variables in Sass to try and keep consistent spacing. 

The variables are set in this bower package: https://github.com/guardian/guss-grid-system/blob/master/_grid-system.scss (but tl;dr, `$gs-baseline` is `16px`, `$gs-gutter` is `20px`).

So if you need 8 vertical pixels, you would use `$gs-baseline * (2/3)`. It's worth noting though that if you're not just halving, doubling etc, it's a bit of a code/design smell (though obviously not always, if you need 8 then you need 8).